recipes/libiconv/all/conanfile.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class LibiconvConan(ConanFile):
+    name = "libiconv"
+    version = "1.15"
+    description = "Convert text to and from Unicode"
+    url = "https://github.com/bincrafters/conan-libiconv"
+    homepage = "https://www.gnu.org/software/libiconv/"
+    author = "Bincrafters <bincrafters@gmail.com>"
+    topics = "libiconv", "iconv", "text", "encoding", "locale", "unicode", "conversion"
+    license = "LGPL-2.1"
+    exports = ["LICENSE.md"]
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {'shared': False, 'fPIC': True}
+    short_paths = True
+    _source_subfolder = "source_subfolder"
+
+    @property
+    def _is_mingw_windows(self):
+        return self.settings.os == 'Windows' and self.settings.compiler == 'gcc' and os.name == 'nt'
+
+    @property
+    def _is_msvc(self):
+        return self.settings.compiler == 'Visual Studio'
+
+    def build_requirements(self):
+        if tools.os_info.is_windows:
+            if "CONAN_BASH_PATH" not in os.environ:
+                self.build_requires("msys2/20161025")
```

Comment:
Is msys2 already contributed 😕❓

Reply:
it's here: https://github.com/conan-io/conan-center-index/tree/master/recipes/msys2/20161025